cmd/mutate-image/main.go

```diff
@@ -0,0 +1,283 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Let's give credit to the people who wrote crane here.

```suggestion

// The algorithm to mutate the image was inspired by https://github.com/google/go-containerregistry/blob/main/cmd/crane/cmd/mutate.go
```